src/components/Greeting.jsx

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from "react";
+
+export const Greeting = () => {
+  const [greeting, setGreeting] = useState("");
+
+  useEffect(() => {
+    const greetTime = new Date().getHours();
+    if (greetTime < 4) setGreeting("Good Night");
+    else if (greetTime < 11) setGreeting("Good Morning");
+    else if (greetTime < 16) setGreeting("Good Afternoon");
+    else if (greetTime < 19) setGreeting("Good Evening");
+    else setGreeting("Good Night");
```

Comment:
Can use switch here

Reply:
I was considering switch-case first but then the lines of code increased, so I decided to stick with the if-else ladder.